src/Features/LanguageServer/CommonLanguageServerProtocol.Framework/ILifeCycleManagement.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace CommonLanguageServerProtocol.Framework;
+
+public interface ILifeCycleManager
```

Comment:
Why is this here? If the concrete is here, and this isn't returned from anything on `LanguageServer` then it seems unnecessary to me.